### ISSUE TITLE
enexSources does not allow multiple .enex files 

### ISSUE THREAD

**Issue by ecerulm** [akosbalasko/yarle#464]

**Comment by akosbalasko** [comment]
hi @ecerulm ,

Pls try to copy your enex files into a directory and set that directory as enexSources. In this cade Yarle should iterate over the contente of the directory and convert all the enex files in it.

**Comment by ecerulm** [comment]
yes, I already did that to workaround it but the https://github.com/akosbalasko/yarle/blob/62d4e19b37f2b09fe8c927490eb702e296425076/README.md?plain=1#L147
 should be updated to say that it's either a single ENEX file or a list of folders, the current `multiple enex files, or a folder of the enex files` seems misleading.

**Comment by akosbalasko** [comment]
@ecerulm okok, thank you, I'll fix it.
